modules/tensor_mechanics/src/materials/ComputeMultipleInelasticCosseratStress.C

```diff
@@ -35,7 +35,8 @@ ComputeMultipleInelasticCosseratStress::ComputeMultipleInelasticCosseratStress(
 void
 ComputeMultipleInelasticCosseratStress::initQpStatefulProperties()
 {
-  ComputeMultipleInelasticStress::initQpStatefulProperties();
```

Comment:
I added this flag to skip over the check for a constant elasticity tensor flag that I added into ComputeMultipleInelasticStress.  With the inheritance as it initially was, I was seeing test failures in several of the jacobian tests (cto29, cwpc01, cwpc02, cdpc01, cdpc02, cdp_cwp_coss01 & 02, coss_elastic) and csv diffs in the capped_weak_plane/small_cosserat tests.  I hadn't tried adding the flag to ComputeCosseratElasticityTensor though, so I'll give that a try.